pkg/littlefs2/Kconfig

```diff
@@ -54,4 +54,11 @@ config LITTLEFS2_BLOCK_CYCLES
         Sets the maximum number of erase cycles before blocks are evicted as a part
         of wear leveling. -1 disables wear-leveling.
 
+config LITTLEFS2_MINIMUM_BLOCK_SIZE
+    int "Minimum acceptable block size"
+    default 512
+    help
+        Sets the minimum acceptable block size which must be a power of two.
```

Comment:
Isn't it safer to express this as the exponent of 2^n? Then we don't have to do any checks

Reply:
Yes, let´s do it like that.

Reply:
And let´s have a special value that sets the block size to the sector size and chose this value as default. So nobody will be forced to reformat an existing FS. However the tests must also pass with different block sizes of course.

Reply:
> special value

let´s choose `< 0`.